ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_activation.py

```diff
@@ -383,6 +383,14 @@ def test_paddle_celu(
     )
 
 
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.rrelu",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+)
+def test_paddle_rrelu(*, dtype_and_x, on_device, fn_tree, frontend, test_flags):
+
 # tanhshrink
 @handle_frontend_test(
```

Comment:
Hi, just thought to hop in, as these are causing base failures. You are using the test body of the `tanhshrink` function. For more clarity, please see: https://github.com/unifyai/ivy/blob/37a63e8305f47e6355b14af04a106b24840d65b8/ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_activation.py#L386-L416

Reply:
cc'ing: @Adeola23 and @soma2000-lang! Thanks!

Reply:
Thanks for bringing this up @khushi-411, Locally this was not the case. I am going create a PR resolving this. Do you know what might have cause this

Reply:
No worries. I think this could have happened while resolving the merge conflicts or updating the branch. Please feel free to ping me if you'd like that PR to get reviewed. Thanks!